### PR TITLE
Clean Tracker layers

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: src/layers/Tracker.js [hunk 1]
```diff
@@ -34,14 +35,18 @@ export default class Tracker {
...
-    this.renderCompleteRef = this.map.once('rendercomplete', () => {
-      [this.canvas.width, this.canvas.height] = this.map.getSize();
-      this.map.getTarget().appendChild(this.canvas);
-    });
-
-    this.changeSizeRef = this.map.on('change:size', () => {
-      [this.canvas.width, this.canvas.height] = this.map.getSize();
-    });
+    // Array of ol events key. We don't use a class property to be sure
+    // it's not overrided by a descendant of this class.
```
Comment (edellucien) [inline src/layers/Tracker.js]:
```suggestion
    // it's not overwritten by a descendant of this class.
```

File: src/layers/Tracker.js [hunk 2]
```diff
@@ -34,14 +35,18 @@ export default class Tracker {
...
+    this.olEventsKeys = [
```
Comment (edellucien) [inline src/layers/Tracker.js]: Maybe stupid question, but why calling it 'olEventsKeys', if it is an array of Event.
Isn't 'olEvents' sufficient?

Comment (oterral) [reply src/layers/Tracker.js]: because they are keys not events

Comment (oterral) [reply src/layers/Tracker.js]: From the doc:
```
Returns:
Unique key for the listener. If called with an array of event types as the first argument, the return will be an array of keys.
```

Comment (oterral) [reply src/layers/Tracker.js]: `olListenersKeys`  is maybe more appropriate.

Comment (edellucien) [reply src/layers/Tracker.js]: Right, then it is the good term. Sorry :)

**Pull request merged**